lib/resources/dstu2/contract.yaml

```diff
@@ -0,0 +1,56 @@
+---
+name: Contract
+field_name_base_url: http://hl7.org/fhir/DSTU2/contract-definitions.html#Contract
+fields:
+
+- name: type
+  type: CodeableConcept
+  binding:
+    description: List of overall contract codes.
+    terminology:
+    - display: SNOMED CT
+      system: http://snomed.info/sct
+      values:
+      - 371537001 - Consent report (record artifact)
+
+- name: subtype
```

Comment:
The link being generated isn't valid, and ends up linking to the top of the Contract page